package.json

```diff
@@ -47,7 +47,7 @@
     "doc": "metadoc -c"
   },
   "engines": {
-    "node": ">=8.6.0"
+    "node": ">=10.16.0"
```

Comment:
Why not 10.0.0 though?

Reply:
`10.15.3` gives `Array.prototype.values()` support, returning `ArrayIterator`
`10.16.0` gives `await events.once(ee, 'eventName')` support